docs/authoring-json-rules.md

```diff
@@ -1,8 +1,8 @@
-# Authoring Template BPA JSON Rules
+# Authoring Template Analyzer JSON Rules
 <a name="note"></a>***Note**: All features that have yet to be implemented have been flagged with an asterisk [\*].*
```

Comment:
A reminder to change the links to the file renamed, there's at least one in this file